### PR TITLE
Format Python code on push automatically

### PR DESCRIPTION
**Pull request by antoinealb**

### CODE REVIEW

File: can-io-firmware/send_pwm.py [hunk 1]
```diff
@@ -7,23 +7,21 @@
 import uavcan
 import os
 
-DSDL_DIR = os.path.join(os.path.dirname(__file__), '../uavcan_data_types/cvra')
+DSDL_DIR = os.path.join(os.path.dirname(__file__), "../uavcan_data_types/cvra")
```
Comment (SyrianSpock) [inline can-io-firmware/send_pwm.py]: Why double quote when you can single quote?

File: tools/arm_graph/parser.py [hunk 2]
```diff
@@ -7,49 +7,53 @@
 def parse(file):
```
Comment (SyrianSpock) [inline tools/arm_graph/parser.py]: Can we delete this file?

File: tools/config/config_to_c.py [hunk 3]
```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env python3
```
Comment (SyrianSpock) [inline tools/config/config_to_c.py]: Do we still need this file?

Comment (antoinealb) [reply tools/config/config_to_c.py]: I dont know. That is not in the scope of this PR in my opinion.

Comment (SyrianSpock) [reply tools/config/config_to_c.py]: True, but still wondering

File: can-io-firmware/send_pwm.py (hunk 1 above)
Comment (antoinealb) [reply can-io-firmware/send_pwm.py]: We just need to pick one and stick to it. So far it is mixed, even in a single file.

File: tools/config/config_to_c.py (hunk 3 above)
Comment (antoinealb) [reply tools/config/config_to_c.py]: I don't think for that one. Maybe for the parser in the other comment.

**Pull request merged**